### ISSUE TITLE
Logarithmic axis problem

### ISSUE THREAD

**Issue by ChrisDimitrov** [oxyplot/oxyplot#571]
Greetings.

I have an issue with the LogarithmicAxis in OxyPlot.WinForms.

I have a plot with a X and Y axis  and several series (usually 2-3 or more) and I want to have the ability to show the Y axis in logarithmic scale.
I tried using the LinearAxis and LogarithmicAxis and switch between them with the "IsAxisVisible" flag, but that does not seem to work.

I also tried to plot the Logarithmic and Linear axis separately, but it seems that the Logaritmic axis only changes the Y scale, and does not redraw the series. Thus, the linear and logarithmic scales are plotted exactly the same way.

How can I achieve the desired functionalliy? Most of the charting libraries provide a "IsLogarithmic" flag that automatically switches the chart to logarithmic scale.

**Comment by objorke** [comment]
did you set the `Key` property on the axes? did you select the `YAxisKey` on the series?
https://github.com/oxyplot/oxyplot/blob/develop/Source/OxyPlot/Axes/Axis.cs#L315
https://github.com/oxyplot/oxyplot/blob/develop/Source/OxyPlot/Series/XYAxisSeries.cs#L91
Clone the source code, use "find usages" in VS on these properties and you should find some examples.

**Comment by ChrisDimitrov** [comment]
Yes, I did set both of these properties, but the problem persists.

**Comment by objorke** [comment]
Please include a code snippet to reproduce the problem.
Is it possible to reproduce as a static plot model? Then it can be included in the example library.

**Comment by ChrisDimitrov** [comment]
Greetings.

You can see the code here: https://github.com/STSSoft/DatabaseBenchmark/tree/master/DatabaseBenchmark/Charts/Linux

In general, I want to use your OxyPlot library in the Database Benchmark application, which is an open surce tool for database performance evaluation.

I am using MS Charts in the Windows version of the application, but I cannot use them under Linux and the Mono Framework. 

Your library seems to provide all of the needed features and if you could provide some implementation with logarithmic axis that works, I would be very grateful.

**Comment by objorke** [comment]
I had a quick look at your code - why do you use "Linear" as key for both the linear and log axes?

**Comment by ChrisDimitrov** [comment]
Because this is a test code and I tried all sorts of variations.

If you want you can provide an implementation suitable for my use case and commit it to Database Benchmark.

When the IsLogarithmic flag is true, the series must be converted to logarithmic axis.
